### DISCUSSION TITLE
Support thumbnail images for blog posts

### DISCUSSION THREAD

**Discussion by peterschmidt85** [squidfunk/mkdocs-material#5736] [Ideas]
Browsing a blog with thumbnail images can be a lot more fun! Many of the companies I admire have already implemented this feature. Here are some great examples:

- Hugging Face's blog: https://huggingface.co/blog
- Supabase's blog: https://supabase.com/blog
- OpenAI's blog: https://openai.com/blog

I believe it would be fantastic if `mkdocs-material` also supported thumbnail images. What do you think? Let me know your thoughts! 😊

**Comment by squidfunk** [comment]
Thanks for the suggestion! We currently try to get the plugin as such stable, but we'll add alternative index page templates in the future. If you want, you can already do it via [customization](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#overriding-templates), as the blog plugin uses the same template system.

**Reply by brospars** [reply]
It works adding the image before the `<!-- more -->` and styling it using css but breaks with pagination
